test/integrationtests/voight_kampff/test_setup.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2020 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import argparse
+from argparse import RawTextHelpFormatter
+from glob import glob
+from os.path import join, dirname, expanduser, basename
+from pathlib import Path
+from random import shuffle
+import shutil
+import sys
+
+import yaml
+
```

Comment:
remove blank line.  all third party software goes in the same section, even if said software was written by us